modules/cnf-topology-aware-lifecycle-manager-PAO-update.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+// Epic CNF-4767 (4.11), Story TELCODOCS-593
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talm-PAO-update_{context}"]
+= Removing Performance Addon Operator subscriptions from deployed clusters
+
+In earlier versions of {product-title}, the Performance Addon Operator provided automatic, low latency performance tuning for applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator.
+
+Do not install the Performance Addon Operator on clusters running {product-title} 4.11 or later. When you upgrade to {product-title} 4.11, the Node Tuning Operator automatically removes the Performance Addon Operator. However, you  need to manually remove any Performance Addon Operator subscriptions to prevent a reinstallation of the Operator. The reference DU profile includes the Performance Addon Operator in the `common-ranGen.yaml` `PolicyGenTemplate`. To remove the subscription from deployed spoke clusters, you must update `common-ranGen.yaml`.
```

Comment:
```suggestion
Do not install the Performance Addon Operator on clusters running {product-title} 4.11 or later. When you upgrade to {product-title} 4.11, the Node Tuning Operator automatically removes the Performance Addon Operator. However, you  need to manually remove any policies that create Performance Addon Operator subscriptions to prevent a reinstallation of the Operator. The reference DU profile includes the Performance Addon Operator in the `common-ranGen.yaml` `PolicyGenTemplate`. To remove the subscription from deployed spoke clusters, you must update `common-ranGen.yaml`.
```